Add "sum" mode

andrewgee opened the pull request:
#### Purpose
Allows values of influxdb query to be summed for the check

andrewgee committed:
Add "sum" mode to check-influxdb-query check

majormoses commented:
Thanks for your contribution to Sensu plugins! Without people like you submitting PRs we couldn't run the project. I will review it shortly.

majormoses commented on bin/check-influxdb-query.rb:
```diff
@@ -114,7 +114,7 @@ class CheckInfluxdbQuery < Sensu::Plugin::Check::CLI
          long: '--mode MODE',
          default: 'first',
-         description: 'How the results are being checked (one of "first", "last", "max", "min", "avg") when the query returns more than one value'
+         description: 'How the results are being checked (one of "first", "last", "max", "min", "sum", "avg") when the query returns more than one value'
```
Makes sense to me, we should probably configure the list of acceptable inputs so that if someone typo's something or tries a function not yet supported they will get better and quicker feedback. Also rather than relying on implicit logic driven case statement it's probably just better to default to `first`
```suggestion
         description: 'How the results are being checked (one of "first", "last", "max", "min", "sum", "avg") when the query returns more than one value',
         in: %w(first last max min sum avg average),
         default: 'first'
```

andrewgee replied:
Sounds good. Applied the suggestion :)

majormoses reviewed (commented):
Looks good to me after addressing my comments about making it a bit better UX.

majormoses committed:
Apply suggestions from code review

Co-Authored-By: andrewgee <andrew@andrewgee.net>

majormoses committed:
minor fixups

Changes:
- remove duplicated default option for `--mode`
- change the else to be an explicit check for first rather than a catch all

Signed-off-by: Ben Abrams <me@benabrams.it>

majormoses commented:
@andrewgee sorry I made a mistake on part of it and was not clear as to what I meant by removing the `case` for `else` which should have been `first`. I pushed the changes to your fork and once it passes CI I will merge and release it.

majormoses commented:
released: https://rubygems.org/gems/sensu-plugins-influxdb/versions/1.4.0
